phylogeny/prototypeSelection.py

```diff
@@ -511,7 +519,9 @@ def prototype_selection_constructive_maxdist(dm: DistanceMatrix,
     # the element to remove first is the one that has smallest distance to all
     # other. "Removing" works by tagging its distance-sum as infinity. Plus, we
     # decrease the number of available elements by one.
-    minElmIdx = currDists.argmin()
+    minElmIdx = next(x for x in currDists.argsort()
```

Comment:
Wait wait wait. We discussed that and agreed that results will most likely be different, but we have not evaluated if the differences are significant. Therefore, we need to run some evaluation. You should be able to do so now - even if slow - with both versions. Just pick 50 out of 1000 elements with 50 preset (so 100 total) and check the sumdist() values for both results.